index.js

```diff
@@ -115,22 +125,22 @@ function configureBot(bot) {
     }
 
     // When the Bot spawns, begin the main gathering loop.
-    // Before collecting a log, have the Bot craft axes if it has none.
+    // Before collecting, have the Bot craft axes if it has none.
     bot.on('spawn', async () => {
         bot.chat('Hello, I have arrived!');
 
-        let logsCollected = bot.getInventoryItemQuantity('spruce_log');
+        let oreCollected = bot.getInventoryItemQuantity('coal_ore');
         let applesCollected = bot.getInventoryItemQuantity('apple');
-        while (logsCollected + applesCollected < 100) {
-            if (!bot.inventoryContainsItem('_axe', {partialMatch: true})) {
-                // craft axes if inventory doesn't have any
+        while (oreCollected + applesCollected < 100) {
+            if (!bot.inventoryContainsItem('_pickaxe', {partialMatch: true})) {
+                // craft pickaxes if inventory doesn't have any
                 await craftAxes();
             }
-            await gatherLog();
+            await gatherCoal();
```

Comment:
Rename to `craftPickAxes` for clarity

Reply:
done, and made it actually craft pickaxes... :D